contributing_to_docs/doc_guidelines.adoc

```diff
@@ -344,15 +320,15 @@ See
 link:https://github.com/openshift/openshift-docs/pull/13878/files#diff-615ba1bf3b09d11a9c2604b775aa32f2[an
 example] of how this is applied.
 
-== Writing Concepts
-A _concept_ is a topic (full .adoc file) or section (individual heading within a
-topic) that supports the things that users want to do and should not include
-task information like commands or numbered steps. Consider topic/section titles
+== Writing concepts
+A _concept_ is an assembly (full .adoc file) or section (individual module within an
+assembly) that supports the things that users want to do and should not include
+task information like commands or numbered steps. Consider section titles
 with a verb like "Understanding <concept>" if it is solely concept-based.
 
-== Writing Tasks
-A _task_ is a topic (full .adoc file) or section (individual heading within a
-topic) that supports the things that users want to do and includes procedural
+== Writing tasks
+A _task_ is an assembly (full .adoc file) or section (individual module within an
+assembly) that supports the things that users want to do and includes procedural
 information like commands and numbered steps. Write tasks in the following
 format.
 
```

Comment:
IDK if you want to be explicit about the format of the task module here or if you want to add a link out to the sample task module in the mod docs guide.